communication/slack_channels.md

```diff
@@ -0,0 +1,62 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels).
+
+### General channels
+
+* `#general`
+* `#offtopic`
```

Comment:
Add description along the lines of:
  * Almost anything goes, the place to share interesting or funny things and keep other channels clean and on topic

Reply:
+1